homeassistant/components/ezviz/sensor.py

```diff
@@ -2,9 +2,13 @@
 from __future__ import annotations
 
 from homeassistant.components.binary_sensor import DEVICE_CLASS_MOTION
```

Comment:
I am not sure if it was a good idea to use device classes from binary_sensor, so I left it as DEVICE_CLASS_MOTION for now.
```suggestion
from homeassistant.components.binary_sensor import BinarySensorDeviceClass
```

Reply:
`motion` is not a valid device class for a sensor, so this is actually a bug.

Reply:
I have removed the device class for the corresponding sensor